### PR TITLE
update octopus web

### PR DESCRIPTION
**Pull request by ksyeo1010**

### CODE REVIEW

File: binding/web/README.md [hunk 1]
```diff
@@ -66,70 +66,48 @@ run:
...
-#### Init options
+#### Octopus Model
 
 Octopus saves and caches your model file in IndexedDB to be used by WebAssembly. Use a different `customWritePath` variable
-to hold multiple models and set the `forceWrite` value to true to force re-save a model file. Set `enableAutomaticPunctuation`
-to false, if you do not wish to enable capitalization and punctuation in transcription.
+to hold multiple models and set the `forceWrite` value to true to force re-save a model file.
 If the model file (`.pv`) changes, `version` should be incremented to force the cached model to be updated.
 
+Either `base64` or `publicPath` must be set to instantiate Octopus. If both are set, Leopard will use the `base64` model.
```
Comment (ErisMik) [inline binding/web/README.md:75]:
```suggestion
Either `base64` or `publicPath` must be set to instantiate Octopus. If both are set, Octopus will use the `base64` model.
```

File: binding/web/README.md [hunk 2]
```diff
@@ -66,70 +66,48 @@ run:
...
-  customWritePath: "octopus_model",
-  forceWrite: false,
-  version: 1
+const octopusModel = {
+  publicPath: ${MODEL_RELATIVE_PATH}, // or
+  base64: ${MODEL_BASE64_STRING},
+  customWritePath: "octopus_model", // Optional
+  forceWrite: false, // Optional
+  version: 1, // Optional
 }
```
Comment (ErisMik) [inline binding/web/README.md:84]:
```suggestion
const octopusModel = {
  publicPath: ${MODEL_RELATIVE_PATH},
  // or
  base64: ${MODEL_BASE64_STRING},
  
  // Optionals
  customWritePath: "octopus_model",
  forceWrite: false,
  version: 1, 
}
```
Just a suggestion

File: binding/web/src/octopus_worker.ts [hunk 3]
```diff
@@ -134,12 +84,19 @@ export class OctopusWorker {
...
+   * @param model.customWritePath Custom path to save the model in storage.
+   * Set to a different name to use multiple models across `octopus` instances.
+   * @param model.forceWrite Flag to overwrite the model in storage even if it exists.
+   * @param model.version Octopus model version. Set to a higher number to update the model file.
    * @param options Optional configuration arguments.
    *
    * @returns An instance of OctopusWorker.
    */
-  private static async create(accessKey: string, modelPath: string, options: OctopusOptions): Promise<OctopusWorker> {
+  private static async create(accessKey: string, model: OctopusModel, options: OctopusOptions): Promise<OctopusWorker> {
```
Comment (ErisMik) [inline binding/web/src/octopus_worker.ts:96]:
```suggestion
  public static async create(accessKey: string, model: OctopusModel, options: OctopusOptions): Promise<OctopusWorker> {
```

File: binding/web/src/types.ts [hunk 4]
```diff
@@ -9,15 +9,14 @@
...
-export type OctopusOptions = {
-  /** @defaultValue 'octopus_model' */
-  customWritePath?: string;
-  /** @defaultValue false */
-  forceWrite?: boolean;
-  /** @defaultValue 1 */
-  version?: number;
-}
+/**
+ * LeopardModel types
```
Comment (ErisMik) [inline binding/web/src/types.ts:15]:
```suggestion
 * OctopusModel types
```

**Review by ErisMik — APPROVED** [review]

File: binding/web/src/octopus.ts [hunk 5]
```diff
@@ -201,15 +150,31 @@ export class Octopus {
    * it can create an instance.
    *
    * @param accessKey AccessKey obtained from Picovoice Console (https://console.picovoice.ai/)
-   * @param modelPath Path to the model saved in indexedDB.
+   * @param model Octopus model options.
+   * @param model.customWritePath Custom path to save the model in storage.
+   * Set to a different name to use multiple models across `octopus` instances.
+   * @param model.forceWrite Flag to overwrite the model in storage even if it exists.
+   * @param model.version Octopus model version. Set to a higher number to update the model file.
```
Comment (laves) [inline binding/web/src/octopus.ts:157]: see my comments on Leopard about this

**Review by laves — CHANGES_REQUESTED** [review]

**Review by laves — APPROVED** [review]

**Pull request merged**